templates/_macros/tiered-list.jinja

```diff
@@ -1,17 +1,13 @@
 # Params
-# is_description_full_width_on_desktop: whether description has its own row on
-#   desktop vs. 50/50 split between title/description
-# is_list_full_width_on_tablet: whether list title/description each have their
-#   own row on tablet vs. 50/50 split between title/description
+# variant: integer representing layout variant; see docs for options:
+# https://vanillaframework.io/docs/patterns/tiered-list
 # Slots
 # title: top-level title element
 # description: top-level description element
 # list_item_title_[1-25]: title element of each child list item
 # list_item_description_[1-25]: description element of each child list item
 # cta: CTA block element
-{% macro tiered_list(
-  is_description_full_width_on_desktop=true,
-  is_list_full_width_on_tablet=true) -%}
+{% macro tiered_list(variant=1) -%}
```

Comment:
What happens if the user passes in an invalid `variant`? Is it a valid default layout?